docs/setup/layers/operation.md

```diff
@@ -0,0 +1,23 @@
+## Quantity operations
+
+The `layer.operation` tag lists operations on quantities in the layer.
+Valid options include:
+
+`GENERATOR`
+: Operation for generating the quantity in the lattice
+
+`DIFFUSER`
+: Operation for diffusion the quantity in the lattice
```

Comment:
this sentence here doesn't make much sense -- diffusivity of the molecule in the lattice ?